qm-plugins/qm-db-connections/class-qm-db-connections-collector.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+class QM_DB_Connections_Collector extends QM_Collector {
+
+	/**
+	 * @var string
+	 */
+	public $id = 'qm-db-connections';
+
+	/**
+	 * @return string
+	 */
+	public function name() {
+		return esc_html__( 'DB Connections', 'query-monitor' );
+	}
+
+	/**
+	 * @return void
+	 */
+	public function process() {
+		foreach ( $GLOBALS as $global ) {
```

Comment:
I know it's a copy-pasta but this to me reads weird? It seems our intent is to check the global $wpdb instance, why don't we do just that rather than iterating over all globals.

Seems like a single  `isset( $GLOBALS['wpdb'] ) && property_exists( $GLOBALS['wpdb'] ,'db_connections' )` would've been just fine?